### ISSUE TITLE
Set initial value of progress

### ISSUE THREAD

**Issue by mauricew** [aitboudad/ngx-loading-bar#68]
I would like to be able to set the initial value for the progress bar, which is currently hardcoded to 2%. Most of the time I set its initial value to 10 so there's more visibility when a request is made.

Maybe making the "set" function public could be done, so that it could be set to any value at any time.

**Comment by aitboudad** [comment]
with #71, `set` and `increment` is public also start accept now passing the initial value:

```ts
// start with 10%
loader.start(10);while starting

// set to 50%
loader.set(50);

// increment by 10%
loader.increment(10);
```

**Comment by aitboudad** [comment]
released in `1.5.0` version.

**Comment by mauricew** [comment]
Thanks! 😁
